test/extended/operators/operators.go

```diff
@@ -156,6 +182,56 @@ var _ = g.Describe("[Feature:Platform][Smoke] Managed cluster should", func() {
 			e2e.Failf("None of the required core operators are available")
 		}
 	})
+
+	g.It("should iterate through operator pods and detect higher-than-normal queries per second", func() {
+		podURLGetter := &portForwardURLGetter{
+			Protocol:   "https",
+			Host:       "localhost",
+			RemotePort: "8443",
+			LocalPort:  "37587",
+		}
+
+		namespaces, err := oc.AdminKubeClient().CoreV1().Namespaces().List(metav1.ListOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		failures := []error{}
+		failedPods := []v1.Pod{}
+		for _, ns := range namespaces.Items {
```

Comment:
Please, please, please, just write a prometheus query that gives you the average over time of QPS.  That would be a 2 line test.